.github/workflows/helmtagbumper.yaml

```diff
@@ -1,6 +1,15 @@
 name: Update API Helm Image Tags
 on:
-  workflow_dispatch: {}
+  workflow_dispatch:
+    inputs:
+      notify-slack:
+        default: true
+        type: boolean
+  workflow_call:
+    inputs:
+      notify-slack:
+        default: false
```

Comment:
Double checking if the expectation is that slack is notified by default on `workflow_dispatch` but not `workflow_call`.

Reply:
Yes, thanks for checking - I [talk about it bit here](https://github.com/DataBiosphere/jade-data-repo/pull/1404#discussion_r1099166863) - Here's what I'm thinking: When it's triggered via workflow_call, it's part of a bigger workflow and that bigger workflow will have its own slack notification. When it's triggered via workflow_dispatch, it's a one-off call, so we want it to have its own slack notification.